inc/namespace.php

```diff
@@ -214,7 +214,7 @@ function ajax_select() : void {
 function replace_attached_file( $file, int $attachment_id ) : string {
 	$attachment = get_post( $attachment_id );
 	if ( ! is_amf_asset( $attachment ) ) {
-		return $file;
+		return ( $file ? $file : '' );
```

Comment:
A boolean might be expected or ok in some instances, it's a little weird but it's hard to make assumptions as to what other assumptions / decisions other plugins make.

The specific thing causing an error is the return value type hint on the function, so we could just remove that and leave the rest of the function as-is.

Reply:
Am I understanding correctly that your suggesting to remove lines 215 - 219:
```php
$attachment = get_post( $attachment_id );
if ( ! is_amf_asset( $attachment ) ) {
	return $file;
}
```

from 

https://github.com/humanmade/asset-manager-framework/blob/83ec67e093831a37c0e46be8d91cc6444ba72c25/inc/namespace.php#L214-L223

Reply:
No, I mentioned the return value type hint. When you write a function e.g. `function my_func() {}` you can specify what type of variable it should return with a return type hint, e.g. `function my_func() : string {}`.

I'm saying we should remove that type hint, and nothing else.